pkg/api/types.go

```diff
@@ -646,15 +648,12 @@ type PodCondition struct {
 	Status ConditionStatus  `json:"status"`
 }
 
-// PodInfo contains one entry for every container with available info.
-type PodInfo map[string]ContainerStatus
-
 // PodContainerInfo is a wrapper for PodInfo that can be encode/decoded
 // DEPRECATED: Replaced with PodStatusResult
 type PodContainerInfo struct {
 	TypeMeta      `json:",inline"`
 	ObjectMeta    `json:"metadata,omitempty"`
-	ContainerInfo PodInfo `json:"containerInfo"`
+	ContainerInfo []ContainerStatus `json:"containerInfo"`
```

Comment:
I've changed this also internally due to https://github.com/GoogleCloudPlatform/kubernetes/issues/3622#issuecomment-76346910